examples/singletap/singletap.ino

```diff
@@ -63,8 +63,8 @@ void loop(void)
 
   while (g_tap_count) {
       Serial.println("Single tap detected!");
-      /* Clear the interrupt by check the source register.. */
-      uint8_t source = accel.checkInterrupts();
+      /* Clear the interrupt as a side-effect of reading the interrupt source register.. */
+      (void)accel.checkInterrupts();
```

Comment:
Is the void cast needed here? Is the CI OK with just a straight call?
```cpp
accel.checkInterrupts();
```

Reply:
right. but it passes just fine without so let's merge it that way.